blazor/dropdown-list/style.md

```diff
@@ -123,4 +145,115 @@ Use the following CSS to customize the appearance of popup element.
     font-family: emoji;
     min-height: 29px;
 }
-```
\ No newline at end of file
+```
+
+## Change the HTML attributes
+
+You can add the additional html attributes such as styles, class, and more to the root element using [HTMLAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_HtmlAttributes) property
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/html-attributes.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with different font family](./images/style/blazor_dropdown_html-attributes.png)
+
+## Set the various font family for dropdown list elements
+
+You can override using the selector for changing the font-family of dropdown list. Also, you can override using the specific class name through [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property.
+
+In the below sample changed the font family of Dropdownlist, ListItem in dropdownlist and filterInput text.
```

Comment:
In the following sample, the font family of the Dropdownlist, ListItem in the dropdown list, and filterInput text are changed.